src/win/fs.c

```diff
@@ -263,57 +308,107 @@ INLINE static void uv_fs_req_init(uv_loop_t* loop, uv_fs_t* req,
 }
 
 
-static int fs__wide_to_utf8(WCHAR* w_source_ptr,
-                               DWORD w_source_len,
-                               char** target_ptr,
-                               uint64_t* target_len_ptr) {
-  int r;
-  int target_len;
-  char* target;
-  target_len = WideCharToMultiByte(CP_UTF8,
-                                   0,
-                                   w_source_ptr,
-                                   w_source_len,
-                                   NULL,
-                                   0,
-                                   NULL,
-                                   NULL);
+static int32_t fs__get_surrogate_value(const WCHAR* w_source_ptr,
+    DWORD w_source_len) {
```

Comment:
style nit: align argument lists
```suggestion
                                       DWORD w_source_len) {
```